aeron-driver/src/main/c/aeron_driver_conductor.c

```diff
@@ -2743,7 +2743,19 @@ aeron_rb_read_action_t aeron_driver_conductor_on_command(
     if (result < 0)
     {
         int os_errno = aeron_errcode();
-        int code = os_errno < 0 ? -os_errno : AERON_ERROR_CODE_GENERIC_ERROR;
+        int code;
+        if (os_errno < 0)
+        {
+            code = -os_errno;
+        }
+        else if (AERON_FILEUTIL_ERROR_ENOSPC == os_errno)
+        {
+            code = AERON_ERROR_CODE_STORAGE_SPACE;
+        }
+        else
+        {
+            code = AERON_ERROR_CODE_GENERIC_ERROR;
```

Comment:
Better to initialise with this value then reset based on conditions.

Reply:
Done.